public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -23,7 +24,10 @@ import { TOP_BAR_LEVEL_HEIGHT } from './types';
 export interface Props extends PropsWithChildren<{}> {}
 
 export function AppChrome({ children }: Props) {
-  const { chrome } = useGrafana();
+  const { chrome, keybindings } = useGrafana();
+  const panelAttentionSubscription = appEvents.getStream(SetPanelAttentionEvent).subscribe((event) => {
+    keybindings.setPanelAttention(event.payload.panelId);
+  });
```

Comment:
Should this subscription be defined in keybindingSrv instead of having a setter for panel attention? Then there is no risk of someone settings it from elsewhere. The subscription would be initiated and deconstructed in AppChrome

Reply:
@tskarhed yea, I would move it to the KeybindingSrv constructor 

> The subscription would be initiated and deconstructed in AppChrome

Don't think you need that as KeybingSrv is a singleton @